src/gam/gapi/directory/customer.py

```diff
@@ -26,7 +25,7 @@ def doGetCustomerInfo():
         result = gapi.call(
             cd.domains(),
             'get',
-            customer=customer_id,
+            customer=customer_info['id'],
```

Comment:
_get_customerid() is meant to handle CUSTOMER_ID env variable with or without C. Why undo this?